test-requirements.txt

```diff
@@ -1,6 +1,7 @@
 coverage
 ddt
 edx-lint==0.4.0
+git+https://github.com/cpennington/hypothesis-python@fast-integers#egg=hypothesis
```

Comment:
https://github.com/HypothesisWorks/hypothesis-python/pull/366 makes it so we can collect a bunch more samples. I'm hoping he'll merge it soon.